exporters/jaeger/include/opentelemetry/exporters/jaeger/recordable.h

```diff
@@ -58,6 +58,7 @@ class JaegerRecordable final : public sdk::trace::Recordable
   std::vector<thrift::Tag> Tags() noexcept { return std::move(tags_); }
   std::vector<thrift::Tag> ResourceTags() noexcept { return std::move(resource_tags_); }
   std::vector<thrift::Log> Logs() noexcept { return std::move(logs_); }
+  std::vector<thrift::SpanRef> References() noexcept { return std::move(references_); }
```

Comment:
Seems `SpanRef`s are not attached to `jaeger_span`?

Reply:
Yup! I goofed and missed bringing over a line. Added: https://github.com/open-telemetry/opentelemetry-cpp/pull/1251/files#diff-3fc4e4d4cb58766986df79016e29275bf3712c2fdcda74c598152f42598bf546R43

(and made sure nothing else is missing)